src/common/schedulers/slurm_commands.py

```diff
@@ -69,16 +68,17 @@ class SlurmNode:
     SLURM_SCONTROL_IDLE_STATE = "IDLE"
     SLURM_SCONTROL_DOWN_STATE = "DOWN"
     SLURM_SCONTROL_DRAIN_STATE = "DRAIN"
-    SLURM_SCONTROL_POWERING_DOWN_STATE = "POWERING_DOWN"
-    SLURM_SCONTROL_POWER_STATE = "IDLE+CLOUD+POWER"
+    SLURM_SCONTROL_POWERING_DOWN_STATES = {"POWERING_DOWN", "POWER_DOWN"}
```

Comment:
Can we also include the change in this PR: https://github.com/aws/aws-parallelcluster-node/pull/404/files
Only consider "POWERING_DOWN" as SLURM_SCONTROL_POWERING_DOWN_STATE?

Reply:
Due to the changes on [line 113](https://github.com/chenwany/aws-parallelcluster-node/blob/bdec3eecf63344926e1bd0e83f312b9b72c71580/src/slurm_plugin/slurm_resources.py#L113) from slurm_resources.py in https://github.com/aws/aws-parallelcluster-node/pull/350 doing that seems circular to me. Would you mind elaborating on the advantage of only considering "POWERING_DOWN" as SLURM_SCONTROL_POWERING_DOWN_STATE?